Formula/neix.rb

```diff
@@ -0,0 +1,22 @@
+class Neix < Formula
+  desc "RSS/Atom feed reader for your terminal"
+  homepage "https://github.com/tomschwarz/neix"
+  url "https://github.com/tomschwarz/neix/archive/refs/tags/v0.1.5.tar.gz"
+  sha256 "23d39f0695fe73a737725c8a1266e02a6a173dcaf0a649ba30db02bc3e54a42f"
+  license "GPL-3.0-only"
+
+  depends_on "cmake" => :build
+  uses_from_macos "curl"
+  uses_from_macos "ncurses"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "install", "PREFIX=#{prefix}"
```

Comment:
Is setting `PREFIX` really needed? `std_cmake_args` already sets `CMAKE_INSTALL_PREFIX`.

Reply:
Indeed, from [the documentation](https://rubydoc.brew.sh/Formula#std_cmake_args-instance_method), `std_cmake_args` contains `install_prefix` parameter which defaults to `prefix`.

Thanks

Reply:
Fixed in https://github.com/Homebrew/homebrew-core/pull/84201/commits/6da5d67a2eda517d317420b9fefab3fa3eee48a8